api/gradlew

```diff
@@ -154,19 +156,19 @@ if [ "$cygwin" = "true" -o "$msys" = "true" ] ; then
         else
             eval `echo args$i`="\"$arg\""
         fi
-        i=$((i+1))
+        i=`expr $i + 1`
```

Comment:
are these changes required, or an opportunistic cleanup?

Reply:
This entire file:
`./gradlew wrapper --version=7.6.1`